packages/mapper/src/mapper.ts

```diff
@@ -206,7 +206,7 @@ export default class Mapper {
       {
         method: 'bulkWrite',
         class: 'Collection',
-        db, coll, arguments: { operations, options }
```

Comment:
`operations` here could be a super large array of documents, so let's omit it entirely from logs.